src/features/metrics/aggregate/index.js

```diff
@@ -97,6 +98,7 @@ export class Aggregate extends FeatureBase {
     try {
       if (this.resourcesSent) return
```

Comment:
This flag means that the metrics will only count up to the first unload (which could happen at vis change).  Is this what we want?